client/src/containers/ContactUsPage/style.css

```diff
@@ -0,0 +1,40 @@
+.contact-container {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+}
+
+.img-container {
+    width: 50%;
+    background-color: var(--main-color);
+}
+
+.img-container img {
+  height: 100vh;
+  width: 516px;
+}
+
+.content-container {
```

Comment:
You could make these selectors more specific (and less likely to accidentally affect other components) by adding `.contact-container` in front of all of them.

Reply:
ok .. i will add the parents to each selectors